validator/core/constants.py

```diff
@@ -22,9 +22,8 @@
 TASK_OFFER_ENDPOINT = "/task_offer/"
 SUBMISSION_ENDPOINT = "/get_latest_model_submission/"
 
-
-GET_ALL_DATASETS_ENDPOINT = "https://content.gradients.io/datasets/random"
-GET_ALL_MODELS_ENDPOINT = "https://content.gradients.io/models/random"
```

Comment:
These still point to the prod content service so not dev like last time. no need to change anything before merging.